### PR TITLE
Pytorch notebook 8.1 draft

### PR DESCRIPTION
**Pull request by GZHoffie**

### CODE REVIEW

**Comment by GZHoffie** [comment]
The training is very slow (Throughput about 1,000/sec while the whole dataset length is 200,000) and becomes even slower for later epochs. The loss also hardly goes down. I adjusted batch size but there is no significant change.
